### PR TITLE
Add job board to the website

### PR DESCRIPTION
**Pull request by ericholscher**

### CODE REVIEW

File: docs/about/stay-connected.rst [hunk 1]
```diff
@@ -27,16 +28,25 @@ or moving them there if they develop into something bigger.
...
 
 Blog
 ~~~~~
 
 Check out our :doc:`/blog/index` to see updates from the Community. These will also be sent to the :doc:`/newsletter`.
 
+Twitter
+~~~~~~~
+
+We're pretty active on `Twitter <https://twitter.com/hashtag/writethedocs?f=tweets&vertical=default&lang=en>`_. You can follow the **#writethedocs** hashtag to see community tweets.
```
Comment (plaindocs) [inline docs/about/stay-connected.rst:49]: Do we want to request that people don't tag @writethedocs here?

File: docs/job-board.rst [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+and good luck on the hunt!
+
+If your company builds software,
+and you're looking for some teammates who care about good documentation,
+we've got you covered.
+Sign up as an employer,
+add your job listing,
+and start talking to the documentarians who can help.
+
+Check it our over a https://jobs.writethedocs.org/.
```
Comment (plaindocs) [inline docs/job-board.rst]: Some spelling niggles here

File: docs/about/stay-connected.rst [hunk 3]
```diff
@@ -27,16 +28,25 @@ or moving them there if they develop into something bigger.
...
 Learn more about our :doc:`/slack`.
 
-Twitter
-~~~~~~~
+Job Board
+~~~~~~~~~
 
-We're pretty active on `Twitter <https://twitter.com/hashtag/writethedocs?f=tweets&vertical=default&lang=en>`__ as well, mostly on **#writethedocs**.
+Write the Docs now has a :doc:`/job-board`.
+This is the place to go if you want to hire or be hired by companies in the Write the Docs community.
```
Comment (plaindocs) [inline docs/about/stay-connected.rst]: I'd actually remove 

> in the Write the Docs community

seems limiting for no need. And if companies don't feel part of the community they might not approach

File: docs/about/stay-connected.rst [hunk 4]
```diff
@@ -27,16 +28,25 @@ or moving them there if they develop into something bigger.
...
+
+Promoted postings will also show up in our newsletter,
+so you can also keep track of jobs by subscribing there!
```
Comment (plaindocs) [inline docs/about/stay-connected.rst]: What about something more along the lines of

We publicise promoted job posts in our newsletter, which goes out to XXX recipients.

You'll only see promoted jobs in the newseletter, so keeping track doesn't feel accurate.

File: docs/about/stay-connected.rst (hunk 1 above)
Comment (ericholscher) [reply docs/about/stay-connected.rst:49]: Don't think that will actually help, sadly :)

**Pull request merged**